__tests__/app.test.js

```diff
@@ -357,4 +357,59 @@ describe("app", () => {
         });
     });
   });
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("status: 201 responds with comment newly added to the database", () => {
+      const newComment = {
```

Comment:
We want it so that if a user doesn't already exist, it should return a 404 as user doesn't exist. Comment should only be posted by a user that exists in the database. You can add the test on line 361 to be a 404 and then create a new test for the happy path using one of the existing users from the database :) Give me a shout if you want to explain in more detail